### PR TITLE
Media migration memory consumption

### PR DESCRIPTION
**Pull request by pierallard**
This PR comes from @masmlar in the repository https://github.com/akeneo/pim-community-dev
The file has moved in the next branches, so we have to bring back here.

Complete description of the PR here :
https://github.com/akeneo/pim-community-dev/pull/5159

### CODE REVIEW

File: upgrades/1.3-1.4/common/AbstractMediaMigration.php [hunk 1]
```diff
@@ -115,6 +115,8 @@ public function createFileInfoTable()
      */
     public function storeLocalMedias()
     {
+        $em = $this->container->get('doctrine.orm.entity_manager');
```
Comment (nidup) [inline upgrades/1.3-1.4/common/AbstractMediaMigration.php]: already declared line 126, you can drop this line

File: upgrades/1.3-1.4/common/AbstractMediaMigration.php [hunk 2]
```diff
@@ -125,6 +127,7 @@ public function storeLocalMedias()
 
         $finder      = new Finder();
         $storedFiles = 1;
+        $batch = 500;
         foreach ($finder->files()->followLinks()->in($this->mediaDirectory) as $file) {
             $fileInfo = $storer->store($file, FileStorage::CATALOG_STORAGE_ALIAS);
             $em->clear();
```
Comment (nidup) [inline upgrades/1.3-1.4/common/AbstractMediaMigration.php]: we already clear it for each file but without detaching, with the bulk clear, this one is useless imho, you can drop it. doing  this we'll clear each 500 files and not each file (should be faster but should consume more memory)

**Comment by pierallard** [comment]
@SamirBoulil Because it's easier to start at 1.4 to master than the opposite...
Plus, the upgrade folder moved from pim-community-standard to pim-community-dev

**Comment by pierallard** [comment]
Remarks of @nidup are totally true, this behavior has already been fixed by @jmleroux in https://github.com/akeneo/pim-community-standard/commit/c14e28619a578406df5f374f57c007dd063d9035 

I will bring back this one to pim-community-dev. Thx !

**Pull request closed without merging**